### PR TITLE
Basic Layout

### PR DESCRIPTION
**Pull request by FaridHai**
Created the Basic Layout for the App. 
Navigation is working. 
Background Image will get transparent (for better view) and colored.  
Add Button without function yet.

### CODE REVIEW

File: frontend/src/components/AddIngredients.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import styled from "styled-components/macro";
+import { useState } from "react";
+
+export default function AddIngredients({ onAddClick }) {
+  const [ingredient, setIngredient] = useState("");
+
+  const handeAddClick = () => {
```
Comment (christian-porsch) [inline frontend/src/components/AddIngredients.js:7]: maybe you want to correct your typo?(hande)

File: frontend/src/components/AddIngredients.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import styled from "styled-components/macro";
+import { useState } from "react";
+
+export default function AddIngredients({ onAddClick }) {
```
Comment (christian-porsch) [inline frontend/src/components/AddIngredients.js:4]: I am not quite sure why you are passing props onAddClick here

**Review by christian-porsch — COMMENTED** [review]
very nice. just check the comments

File: frontend/src/components/AddIngredients.js (hunk 1 above)
Comment (FaridHai) [reply frontend/src/components/AddIngredients.js:7]: Thanks for the review. I will correct the Typo.

File: frontend/src/components/AddIngredients.js (hunk 2 above)
Comment (FaridHai) [reply frontend/src/components/AddIngredients.js:4]: Function will be added soon.

**Review by christian-porsch — COMMENTED** [review]
sweet!

**Pull request merged**